### DISCUSSION TITLE
New implementation (Lua Server Pages)

### DISCUSSION THREAD

**Discussion by surfskidude** [ColorlibHQ/AdminLTE#3479] [General]
Hello,

Just a quick note to let you know that I have created a new AdminLTE implementation for Lua Server Pages (LSP)

Source code: https://github.com/RealTimeLogic/LSP-Examples/tree/master/Dashboard
Tutorial: https://makoserver.net/articles/How-to-Build-an-Interactive-Dashboard-App

Thanks

**Comment by danny007in** [comment]
Cool 👍🏼 
After verification
we will add to docs

**Reply by REJack** [reply]
@danny007in can you add it before the release?

**Reply by surfskidude** [reply]
Thank you. Also, please let me know if you have any recommendations, comments, etc. on the article.

**Reply by danny007in** [reply]
make release https://github.com/RealTimeLogic/LSP-Examples first @surfskidude

**Reply by surfskidude** [reply]
Can't use "make", but I created an installation script for you:
bash <(curl -s https://raw.githubusercontent.com/RealTimeLogic/LSP-Examples/master/Dashboard/install.sh)

**Reply by danny007in** [reply]
@surfskidude can u make PR for this 
for urs reference https://github.com/ColorlibHQ/AdminLTE/issues/3286

**Reply by surfskidude** [reply]
Not following you. I have nothing to push. I have not made any mods to AdminLTE.

**Reply by danny007in** [reply]
make PR for AdminLTE Docs in https://adminlte.io/docs/3.0/implementations.html

**Reply by surfskidude** [reply]
done
